hugegraph-core/src/main/java/com/baidu/hugegraph/backend/tx/GraphTransaction.java

```diff
@@ -207,6 +207,7 @@ protected BackendMutation prepareCommit() {
 
     protected void prepareAdditions(Map<Id, HugeVertex> addedVertexes,
                                     Map<Id, HugeEdge> addedEdges) {
+        this.checkCustomizedIdVertex(addedVertexes);
```

Comment:
considering that allow the loader to import quickly, we should still add a config option to disable this check.